source/fundamentals/connection/tuning.txt

```diff
@@ -0,0 +1,255 @@
+==================
+Tuning Connections
+==================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _connection-tuning-java:
+
+Overview
+--------
+
+In this guide, you can learn about **connection pools** and **connection storms**.
+
+Connection Pools
+----------------
+
+Connection pools are fundamental to the performance of your application.
+A **connection pool** is a cache of database connections maintained by the driver
+so that future requests can reuse already established connections.
```

Comment:
A couple items here:

1) Try to define a term before we talk about its benefits. The pattern our team has been using which I think aligns with the style guide is to use bold formatting for the first mention; optional if you define it in the same sentence as the first mention. Then, we define the term in the next sentence.
2) Make sure the reader understands how it relates to MongoDB and why it can help performance.

```suggestion
A connection pool is a cache of active connections maintained by the driver.
The driver uses connection pools to improve your application's performance
by running your MongoDB operations on one of its already-established connections.
```